src/components/ui/ComboBox.tsx

```diff
@@ -6,29 +6,43 @@ function classNames (...classes): string {
   return classes.filter(Boolean).join(' ')
 }
 
-interface ValueObj{
+export interface ValueObj{
   id: number
   name: string
+  disabled?: boolean
 }
 
-interface ComboBoxProps{
+interface ComboBoxSingleProps {
   options: ValueObj[]
   value: ValueObj
   onChange: (value: ValueObj) => void
   label: string
+  selectClassName?: string
 }
-export default function ComboBox ({ options, value, onChange, label }: ComboBoxProps): JSX.Element {
+interface ComboBoxMultiProps {
+  options: ValueObj[]
+  value: ValueObj[]
+  onChange: (value: ValueObj[]) => void
+  label: string
+  multiple: true
+  selectClassName?: string
+}
+type ComboBoxProps = ComboBoxSingleProps | ComboBoxMultiProps
```

Comment:
A bit of a hack to reuse the `ComboBox`. But perhaps I have gone too far and should have create a brand new component for multi-selections. Lmk if you think so and I can make the change.

Reply:
I think we're good for now.  If there's a need for multi-select combobox again we can create a reusable one.